granian/cli.py

```diff
@@ -198,6 +198,11 @@ def option(*param_decls: str, cls: Optional[Type[click.Option]] = None, **attrs:
     '--process-name',
     help='Set a custom name for processes (requires granian[pname] extra)',
 )
+@option(
+    '--pid',
```

Comment:
Maybe a little of a *nitpick* here, but I would name the parameter `--pid-file`

Reply:
Addressed via https://github.com/emmett-framework/granian/compare/405ff32e5fbe32a1ac6db956e6d9df2c71913be8..ea4efa7dad1e4b1fbe09d2dfd822936be9d2095d